content/download/fedora.adoc

```diff
@@ -4,14 +4,24 @@ iconhtml = "<div class='fl-fedora'></div>"
 weight = 20
 +++
 
-=== 4.0.2 Stable Release
-KiCad 4.0.2 is available in the Fedora 23 repo.
+=== Stable Release
+In general, latest Fedora releases ship the stable versions KiCad as they are
+released, with a delay of around a week that the package spends in the testing
```

Comment:
Also, I just got the copr build up and running yesterday, would you be able to test some of the build  from there?

https://copr.fedorainfracloud.org/coprs/g/kicad/kicad/

I would like to link to that copr instead. I would appreciate if you can add info on how to remove the old copr and use the new.